beacon_node/execution_layer/src/engine_api.rs

```diff
@@ -222,7 +214,7 @@ impl<T: EthSpec> TryFrom<ExecutionPayload<T>> for ExecutionBlockWithTransactions
                 transactions: block
                     .transactions
                     .iter()
-                    .map(|tx| Transaction::decode(&Rlp::new(tx)))
+                    .map(|tx| TransactionSigned::decode_enveloped(tx.as_ref().into()))
```

Comment:
Have you tested this on the devnet? 
Also, knowing that reth is still in alpha stage, I'm just trying to gauge what's the worst that can happen in the case anything breaks here (no disrespect, those guys are awesome).
In my understanding, this isn't actually consensus breaking for us because we are only using this to reconstruct payloads when serving block requests over rpc or the http api. 
Does this seem right to you?

Reply:
right, it's only used when we are reconstructing payloads from blinded blocks, so it would mean other clients wouldn't be able to sync from lh nodes that prune blinded payloads. We also wouldn't be able to re-construct pruned payloads for http api requests

working on testing now!

Yea I'm not sure which way to go as far as trying to get blob transactions into `ethers` or just jumping to `reth`. Blob transactions will likely be tested more thoroughly earlier in `reth` but obviously other/older transaction types will have been pretty well hardened on `ethers`.  The same team is really behind both, and `reth` types will be well maintained, so I'm sort of in the camp of jumping to `reth`, but don't really mind either way.

Reply:
Damn looks like these deps require rust `1.70`
```
164.4   package `codecs-derive v0.1.0-alpha.6 (https://github.com/paradigmxyz/reth.git?tag=v0.1.0-alpha.6#88aea631)` cannot be built because it requires rustc 1.70 or newer, while the currently active rustc version is 1.68.2
```

Reply:
Ok turns out this is legacy code and we can deprecate it instead (imo we should do that in a pr to unstable)

Reply:
reverted the changes to add `reth-primitives`